release/release_data_tests.yaml

```diff
@@ -477,6 +477,29 @@
       python gpu_batch_inference.py
       --data-directory 10T-image-data-synthetic-raw-parquet --data-format parquet
 
+- name: batch_inference_mock_image_pipeline
+  working_dir: nightly_tests
+
+  cluster:
+    cluster_compute: dataset/autoscalling_100_gpu_compute.yaml
+
+  run:
+    timeout: 3600
+    script: >
+      python dataset/batch_inference_mock_image_pipeline.py
+
+- name: batch_inference_mock_image_pipeline_chaos
+  working_dir: nightly_tests
+
+  cluster:
+    cluster_compute: dataset/autoscalling_100_gpu_compute.yaml
+
+  run:
+    timeout: 3600
+    prepare: python setup_chaos.py --chaos TerminateEC2Instance --batch-size-to-kill 10 --max-to-kill 100 --kill-delay 120
+    script: >
+      python dataset/batch_inference_mock_image_pipeline.py
+
```

Comment:
(Optionally for future reference): we can use the variants syntax to minimize duplication https://github.com/ray-project/ray/blob/251a1315ed44b2aefd4ad85ca7314f85b234dea0/release/release_data_tests.yaml#L211-L232